components/nifosg/controller.cpp

```diff
@@ -385,8 +385,9 @@ void AlphaController::apply(osg::StateSet *stateset, osg::NodeVisitor *nv)
     }
 }
 
-MaterialColorController::MaterialColorController(const Nif::NiPosData *data)
+MaterialColorController::MaterialColorController(const Nif::NiPosData *data, int color)
```

Comment:
Consider use `TargetColor` for `color` type. This makes object construction more explicit. A place where object is constructed may have more context about how to convert `int` to `TargetColor`. Otherwise some explanation why type is hidden and why is any `int` value allowed should be here to avoid mistakes in the future.